website/static/js/contribManager.js

```diff
@@ -356,7 +356,8 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
         self.forceSubmit(true);
         bootbox.confirm({
             title: 'Save changes?',
-            message: 'Are you sure you want to save these changes?',
+            message: 'Are you sure you want to save these changes? If you are removing a contributor with checked-out' +
```

Comment:
The second sentence should only be present if a contributor is being removed (it currently shows for any update to the contributors).

That said, I think it'd be fine to remove that sentence altogether if we think it's consistent with user expectation anyway. Pinging @erinmayhood for feedback.